travis.sh

```diff
@@ -290,6 +302,12 @@ function test_workspace() {
    echo -e $(colorize YELLOW Test blacklist: $(colorize THIN $TEST_BLACKLIST))
    test -n "$TEST_BLACKLIST" && catkin config --blacklist $TEST_BLACKLIST &> /dev/null
 
+   # Consider TEST_WHITELIST
+   TEST_WHITELIST=$(unify_list " ,;" ${TEST_WHITELIST:-})
+   echo -e $(colorize YELLOW Test whitelist: $(colorize THIN $TEST_WHITELIST))
+   test -n "$TEST_WHITELIST" && catkin config --whitelist $TEST_WHITELIST &> /dev/null
+
```

Comment:
With these new lines I was able to split up the code coverage run on moveit over two travis runs and solve the timeout issue.  This is what I put int he moveit `.travis.yml` file:

```
- TEST="code-coverage" TEST_WHITELIST="moveit_core"
- TEST="code-coverage" TEST_BLACKLIST="moveit_ros_perception moveit_core"
```